Documentation/ApiOverview/CachingFramework/Configuration/_default.php

```diff
@@ -0,0 +1,4 @@
+<?php
+
```

Comment:
should we still test for TYPO3 to be set here? At least in non-composer enviroments the file is publicly available?

Reply:
It is checked: the null-coalescing operator is used. So the empty array is only applied when myext_cache is not available.

Reply:
We don't suggest to use the TYPO3 constant as gate keeper:
https://docs.typo3.org/m/typo3/reference-coreapi/12.4/en-us/Configuration/Typo3ConfVars/Index.html#file-config-system-additional-php

This should be the place to adjust, but I am not sure if this is really recommended.